mmv1/templates/terraform/examples/app_engine_standard_app_version_outbound.tf.erb

```diff
@@ -0,0 +1,128 @@
+# Example of setting up a App Engine service with a static outbound IP
+
+# [START appengine_service_static_network]
+resource "google_compute_network" "default" {
+  name = "<%= ctx[:vars]['appengine_static_ip_network'] %>"
+}
+# [END appengine_service_static_network]
+
+# [START appengine_service_static_subnet]
+resource "google_compute_subnetwork" "default" {
+  name          = "<%= ctx[:vars]['appengine_static_ip'] %>"
+  ip_cidr_range = "10.124.0.16/28"
+  network       = google_compute_network.default.id
+  region        = "us-central1"
+}
+# [END appengine_service_static_subnet]
+
+# [START appengine_service_static_vpc_conn]
+resource "google_project_service" "vpc" {
+  service = "vpcaccess.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_vpc_access_connector" "default" {
+  provider = google-beta
```

Comment:
```suggestion
```

Reply:
Provider declarations shouldn't be defined in configurations. If this can only run in beta environment please make the test `min_version : beta`.